docs/reference/eventing/eventing.md

```diff
@@ -8977,4 +8977,4 @@ knative.dev/pkg/apis/duck/v1.SourceStatus
 <p><em>
 Generated with <code>gen-crd-api-reference-docs</code>
 on git commit <code>e018fe79</code>.
-</em></p>
```

Comment:
Did this need to change?

If not, I'd prefer to leave this out, because then @lionelvillard can approve the change.